opta/aws/flyte.yaml

```diff
@@ -89,40 +89,8 @@ modules:
             "eks.amazonaws.com/role-arn": "${{module.adminflyterole.role_arn}}"
       workflow_scheduler:
         enabled: true
```

Comment:
Is the native scheduler the default value?

Reply:
yes

Reply:
It is coming from eks values https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/values-eks.yaml#L329